x-pack/plugins/ingest_manager/common/types/models/output.ts

```diff
@@ -11,6 +11,9 @@ export enum OutputType {
 
 const OutputBaseSchema = {
   name: schema.string(),
+  is_default: schema.boolean({
+    defaultValue: false,
```

Comment:
I don't think default values should be provided for the way these schemas are used